### PR TITLE
Adding snippet tags for callable and task queue sample imports

### PR DESCRIPTION
**Pull request by egilmorez**
I'd like to do this for all the guides, since it seems to be widely requested and not as obvious as one might think.  

Do I have this right for TQ though?  Lots of imports iiuc!

### CODE REVIEW

File: Node/quickstarts/callable-functions/functions/index.js [hunk 1]
```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
+// [START imports]
 const {onCall, HttpsError} = require("firebase-functions/v2/https");
 const {getDatabase} = require("firebase-admin/database");
 const {logger} = require("firebase-functions/v2");
 const sanitizer = require("./sanitizer");
+// [END imports]
```
Comment (jhuleatt) [inline Node/quickstarts/callable-functions/functions/index.js:22]:
```suggestion
// [START imports]
// Dependencies for callable functions
const {onCall, HttpsError} = require("firebase-functions/v2/https");
const {logger} = require("firebase-functions/v2");

// Dependencies for the addMessage function
const {getDatabase} = require("firebase-admin/database");
const sanitizer = require("./sanitizer");
// [END imports]
```

File: Node/taskqueues-backup-images/functions/index.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 "use strict";
+// [START imports]
 const path = require("path");
 const fetch = require("node-fetch");
 const functions = require("firebase-functions");
```
Comment (egilmorez) [inline Node/taskqueues-backup-images/functions/index.js:20]: Remove this 1st-gen import const functions = require("firebase-functions");

File: Node/taskqueues-backup-images/functions/index.js [hunk 3]
```diff
@@ -25,6 +26,7 @@ const {getStorage} = require("firebase-admin/storage");
 const {GoogleAuth} = require("google-auth-library");
 const logger = functions.logger;
 const HttpsError = functions.https.HttpsError;
```
Comment (egilmorez) [inline Node/taskqueues-backup-images/functions/index.js:28]: Strictly for TQ:

const {onTaskDispatched} = require("firebase-functions/v2/tasks");
const {onRequest} = require("firebase-functions/v2/https");
const {getFunctions} = require("firebase-admin/functions");
const logger = functions.logger; (only do this the right way, like in the callable sample)

**Review by egilmorez — COMMENTED** [review]
Thanks for the input Jeff!

Not sure why I think this, but I wanted to add the Admin SDK import to the general/required section.   LMK if that might be valid.

**Comment by egilmorez** [comment]
Oooh yeah, Python . . . adding similar import bifurcation in Python samples and adding Kevin for review.  

Thanks!

File: Python/quickstarts/callable-functions/functions/main.py [hunk 4]
```diff
@@ -15,9 +15,14 @@
 import re
 from typing import Any
 
-from firebase_admin import db, initialize_app
+# [START v2imports]
+# Dependencies for callable functions
 from firebase_functions import https_fn, options
 
+# Dependencies for messageFunctionTrigger
```
Comment (kevinthecheung) [inline Python/quickstarts/callable-functions/functions/main.py:22]: How about "Dependencies for writing to RTDB"?

(There's no messageFunctionTrigger)

File: Python/taskqueues-backup-images/functions/main.py [hunk 5]
```diff
@@ -12,19 +12,23 @@
...
-
 from firebase_admin import initialize_app, storage
 from firebase_functions import https_fn, tasks_fn, params
-from firebase_functions.options import RetryConfig, RateLimits, SupportedRegion
-
 import google.auth
 from google.auth.transport.requests import AuthorizedSession
-from google.cloud import tasks_v2
-import requests
+// [END v2imports]
```
Comment (kevinthecheung) [inline Python/taskqueues-backup-images/functions/main.py:30]:
```suggestion
# [END v2imports]
```

File: Python/taskqueues-backup-images/functions/main.py [hunk 6]
```diff
@@ -12,19 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START v2imports]
+# Dependencies for task queue functions
+from google.cloud import tasks_v2
+import requests
+from firebase_functions.options import RetryConfig, RateLimits, SupportedRegion
+
+// Dependencies for image backup sample
```
Comment (kevinthecheung) [inline Python/taskqueues-backup-images/functions/main.py:21]:
```suggestion
# Dependencies for image backup
```

**Review by kevinthecheung — APPROVED** [review]

File: Python/quickstarts/callable-functions/functions/main.py (hunk 4 above)
Comment (egilmorez) [reply Python/quickstarts/callable-functions/functions/main.py]: Got it!  I think I stole that from a tag name :)

**Pull request merged**